src/views/Home.vue

```diff
@@ -3,23 +3,25 @@
     <Header :image="imageHeader"
             :title="titleHeader"
             :subtitle="subtitleHeader"/>
+
     <Navigation/>
     <Products/>
     <Cart/>
   </div>
 </template>
 
 <script lang="ts">
-import { Options, Vue } from "vue-class-component";
+import {Options, Vue} from "vue-class-component";
```

Comment:
To w jednym pliku używacie compositionAPI a w drugim ClassAPI? Nie lepiej wszystko robić w jednym stylu?